### PR TITLE
resolved merge conflict

### PR DESCRIPTION
**Pull request by favouridoko1**
# Description

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "clsx": "^2.1.1",
     "isbot": "^4.4.0",
     "lucide-react": "^0.408.0",
+    "pnpm": "^9.6.0",
```
Comment (abiona01) [inline package.json]: Remove pnpm from the project. You're supposed to install it on your laptop, not in the project

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (favouridoko1) [reply package.json]: alright ma

File: package-lock.json [hunk 2]
```diff
@@ -10394,6 +10395,21 @@
...
+        "pnpm": "bin/pnpm.cjs",
+        "pnpx": "bin/pnpx.cjs"
+      },
+      "engines": {
+        "node": ">=18.12"
+      },
+      "funding": {
+        "url": "https://opencollective.com/pnpm"
+      }
+    },
```
Comment (abiona01) [inline package-lock.json:10412]: This too. Don't manually delete. Run `pnpm remove pnpm`

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (abiona01) [inline pnpm-lock.yaml:1]: Why did you delete the lock file?

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (favouridoko1) [reply pnpm-lock.yaml:1]: ooh noo

Comment (favouridoko1) [reply pnpm-lock.yaml:1]: just fixed that now
